### PR TITLE
UI - new hosts, step 1 updates

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*

- bcgov/entity/issues/

*Description of changes:*
- updates for fee things
- locale stuff
- address stuff

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Review by deetz99 — APPROVED** [review]
looks good 👍

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-329-113dd7cx.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-329-76l27d3s.web.app

**Review by thorwolpert — APPROVED** [review]

File: strr-host-pm-web/app/locales/en-CA.ts
Comment (dimak1) [inline strr-host-pm-web/app/locales/en-CA.ts:1]: @kialj876 `ts` vs `json` for locales, just curious your opinion?

**Pull request merged**